src/sol_imgui/sol_imgui.h

```diff
@@ -2,6 +2,13 @@
 
 namespace sol_ImGui
 {
+
+// Version
+inline std::string GetVersion()
```

Comment:
There isnt command for this?

Reply:
oh yeah sorry i'll remove this. it's back from when i was doing some initial testing.